doc/source/ray-air/examples/convert_existing_tf_code_to_ray_air.ipynb

```diff
@@ -0,0 +1,1070 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result.metrics_dataframe.plot(\"training_iteration\", [\"train_loss\", \"test_loss\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9754329",
+   "metadata": {},
+   "source": [
+    "### Continue training from a checkpoint\n",
+    "\n",
+    "In the case that your training run gets interrupted (e.g. intentionally interrupted by a user or a preempted by a spot instance), we may want to resume training from a checkpoint.\n",
```

Comment:
For failure handling should we be showing using `FailureConfig` instead to automatically continue training from the latest checkpoint? Rather than doing it manually.

I don't think we should be recommending `resume_from_checkpoint` for fault tolerance purposes

Reply:
Yeah, makes sense. It's hard to show the `FailureConfig` in action, so I'll just write a note about what it's doing.